app/code/PalashAs/AskQuestion/Controller/Adminhtml/Question/MassStatus.php

```diff
@@ -0,0 +1,62 @@
+<?php
+
+namespace PalashAs\AskQuestion\Controller\Adminhtml\Question;
+
+use Magento\Backend\App\Action\Context;
+use PalashAs\AskQuestion\Model\ResourceModel\AskQuestion\CollectionFactory;
+use Magento\Ui\Component\MassAction\Filter;
+use Magento\Framework\Controller\ResultFactory;
+use PalashAs\AskQuestion\Model\ResourceModel\AskQuestion\Collection;
+
+/**
+ * Class MassStatus
+ * @package PalashAs\AskQuestion\Controller\Adminhtml\Question
+ */
+class MassStatus extends AbstractMassAction
+{
+    /**
+     * @var Filter
+     */
+    protected $filter;
+
+    /**
+     * @var CollectionFactory
+     */
+    protected $collectionFactory;
+
+    /**
+     * @param Context $context
+     * @param Filter $filter
+     * @param CollectionFactory $collectionFactory
+     */AbstractMassaction
+    public function __construct(
+        Context $context,
+        Filter $filter,
+        CollectionFactory $collectionFactory
+    ) {
+        $this->filter = $filter;
+        $this->collectionFactory = $collectionFactory;
+        parent::__construct($context);
+    }
+
+    /**
+     * @param Collection $collection
+     * @return mixed
+     */
+    protected function massAction(Collection $collection)
+    {
+        $questionChangeStatus = 0;
```

Comment:
Need Use  "try,  catch for save" and $collection have method $collection->count()